subscription.go

```diff
@@ -196,6 +198,29 @@ func (s *Subscription) sendNotification(ctx context.Context, data *PublishNotifi
 
 }
 
+// Stats returns a diagnostic struct with metadata about the current subscription
+func (s *Subscription) Stats() (*ua.SubscriptionDiagnosticsDataType, error) {
+	// TODO(kung-foo): once browsing feature is merged, attempt to get direct access to the
```

Comment:
does this refer to the more browsing branch or something else?

Reply:
yes. though I haven't tried it on this exact use case. So it is possible that I can't get access to the single diag node. Also, I noticed that with the FreeOPCUA server, no diag nodes are generated. Is there a standard way to indicate that that feature is not supported?

Reply:
I don't know.